lib/turn.rb

```diff
@@ -0,0 +1,22 @@
+class Turn
+attr_reader :guess, 
+            :card, 
+            :feedback
```

Comment:
We don't need the `feedback` `attr_reader` because this method is also defined below on line 15.